Updates to ReadMe for setup instructions and added basic seed scripts

jylee3805 opened the pull request.

JeremiahRobert23 committed:
Merge branch 'dev' into jrobert/crud-operations

JeremiahRobert23 committed:
Merge branch 'JLee/fastAPI-routes' into jrobert/crud-operations

JeremiahRobert23 committed:
Merge remote-tracking branch 'origin/dev' into jrobert/crud-operations

JeremiahRobert23 committed:
Update database.py

JeremiahRobert23 committed:
Merge branch 'JLee/fastAPI-routes' into jrobert/crud-operations

JeremiahRobert23 committed:
Merge remote-tracking branch 'origin/dev' into jrobert/crud-operations

JeremiahRobert23 committed:
my db

my pass

JeremiahRobert23 committed:
Get attributes from url & post to db

JeremiahRobert23 committed:
merge changes

JeremiahRobert23 committed:
train and other changes

JeremiahRobert23 committed:
Merge branch 'dev' into jrobert/crud-operations

JeremiahRobert23 committed:
New method for scraping all images

JeremiahRobert23 committed:
changes to scrape_all_images function

jylee3805 committed:
Added seed scripts

JeremiahRobert23 committed:
Merge remote-tracking branch 'origin/main' into jrobert/crud-operations

jylee3805 committed:
Updating ReadMe

jylee3805 committed:
ReadMe Changes

jylee3805 committed:
ReadMe Changes

jylee3805 committed:
ReadMe Changes

jylee3805 committed:
ReadMe Changes

JeremiahRobert23 committed:
changes to scrape carousel function

The function get all image url from the carousel of given home.com page

kugelblitz104 commented on README.md:
```diff
@@ -2,36 +2,75 @@
+
+### Access API
+
+Open your browser and visit host http://127.0.0.1:8000 to access the API
```
```suggestion
Open your browser and visit host http://localhost:8000 to access the API
```

kugelblitz104 reviewed (approved):
Looks good! Thank you!

kugelblitz104 commented:
I am going to reach out to Syed for his approval as well, don't merge until he has approved for this.

kugelblitz104 commented on README.md:
```diff
@@ -2,36 +2,75 @@
+
+https://github.com/containers/podman/blob/main/docs%2Ftutorials%2Fpodman-for-windows.md
+
+
```
Add something here to install podman compose

JeremiahRobert23 committed:
Update the example route in listings_router.py

Image scrape using scrape_carousel_image function
and some changes to example route

JeremiahRobert23 committed:
Update POST listings/url route

POST route that uses url and each table correctly

jylee3805 committed:
Updated Yolo model and changes to photos router

JeremiahRobert23 committed:
Merge branch 'jrobert/crud-operations' into dev
